<?xml version="1.0"?>
<!--|
    Copyright (C) 2019-2022 Learning Algorithms and Systems Laboratory, EPFL, Switzerland
    Authors:  Konstantinos Chatzilygeroudis (maintainer)
              Matthias Mayr
              Bernardo Fichera
    email:    costashatz@gmail.com
              matthias.mayr@cs.lth.se
              bernardo.fichera@epfl.ch
    Other contributors:
              Yoan Mollard (yoan@aubrune.eu)
              Walid Amanhoud (walid.amanhoud@epfl.ch)
    website:  lasa.epfl.ch

    This file is part of iiwa_ros.

    iiwa_ros is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    iiwa_ros is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

|-->
<launch>
  <!-- Load IK params from YAML file to parameter server -->
  <rosparam file="$(find iiwa_tools)/config/iiwa_service.yaml" command="load"/>

  <!-- launch the IK server -->
  <node name="iiwa_service" pkg="iiwa_tools" type="iiwa_service" respawn="false" output="screen"/>

</launch>